### PR TITLE
Added a fixed navbar

### PR DESCRIPTION
**Pull request by Demiture**
Added a fixed navbar that scrolls through the site via. element ids
also has links to the sites other pages, and is very easy to add too
tried to keep it slim, dark, and out of the way; but kept the green theme as a highlight to hovering the mouse over each button

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -330,6 +330,23 @@ body:not(.loaded) > *:not(#loader-wrapper) {
...
+  {
+    position: fixed;
+    width: 100%;
+    background-color: #212121;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    z-index: 100;
+  }
+
+/* Low-key not sure this is needed but oh well */
```
Comment (T44Nadeko) [inline css/style.css]: "Low-key" == 🙅‍♂️

File: css/style.css [hunk 2]
```diff
@@ -330,6 +330,23 @@ body:not(.loaded) > *:not(#loader-wrapper) {
...
+  #navbar .is-scroll
+  {
+    background-attachment:	scroll,								scroll;
```
Comment (T44Nadeko) [inline css/style.css:345]: but why so many tabs 🤔

File: css/style.css [hunk 3]
```diff
@@ -723,6 +740,28 @@ body:not(.loaded) > *:not(#loader-wrapper) {
 					background: rgba(58,57,57,0.025);
 				}
 
+        input[type="button"].style4,
+  		  input[type="submit"].style4,
```
Comment (T44Nadeko) [inline css/style.css:743]: double space and a tab? *but why*

File: css/style.css [hunk 4]
```diff
@@ -723,6 +740,28 @@ body:not(.loaded) > *:not(#loader-wrapper) {
...
 
+        input[type="button"].style4,
+  		  input[type="submit"].style4,
+  		  input[type="reset"].style4,
+        .button.style4
+        {
+          padding: 2px 8px;
+          text-align: center;
+          color: white;
+          background-color: #212121;
```
Comment (T44Nadeko) [inline css/style.css:750]: so much inconsitencies between tabs and spaces, do a find all and replace the spaces with tabs. N++ can do it if you use the regex mode, replacing with `/t`

File: index.php [hunk 5]
```diff
@@ -32,6 +32,20 @@
...
 	<body class="is-loading">
 
+		<div class="bar" id="navbar">
+			<div class="insidebar">
+				<a href="#banner" class="button style4 scrolly-middle">Connect</a>
+				<a href="#community" class="button style4 scrolly-middle">Community</a>
+				<a href="#first" class="button style4 scrolly-middle">Game Servers</a>
+				<a href="#contributors" class="button style4 scrolly-middle">Contributors</a>
+				<a href="#t44hq" class="button style4 scrolly-middle">T44 HQ</a>
+				||
```
Comment (T44Nadeko) [inline index.php:42]: Make sure to check this document for tabs and space inconsistencies as well

File: index.php [hunk 6]
```diff
@@ -263,8 +277,9 @@
 	        <p>
 		    <center>
 					<img src="https://i.imgur.com/nPxjS4J.gif" />
-			    		<p style="font-family:courier; font-size:60%; color: white;">Cursed Image &copy; Maria 2017</p>
-			</center>
+					<p style="font-family:courier; font-size:60%;">Cursed Image &copy; Maria 2017</p>
+					<iframe width="1280" height="720" src="https://www.youtube.com/embed/96Mbt36ayZ8" frameborder="0" allowfullscreen></iframe>
```
Comment (T44Nadeko) [inline index.php]: Rule #1 of pleasing me: tags must end either on the same line, or with the same indentation a few lines down

**Review by T44Nadeko — CHANGES_REQUESTED** [review]
I'm critical :)

File: css/style.css (hunk 2 above)
Comment (Demiture) [reply css/style.css:345]: This is literally from further down the css file 👀

Comment (fearnlj01) [reply css/style.css:345]: I fix things when I see them, there's a reason I haven't gone near any of the css sheets ¯\\_(ツ)\_/¯

**Comment by Demiture** [comment]
Done. @fearnlj01 🖕

**Pull request merged**